hash/pce_tourvision.xml

```diff
@@ -543,7 +543,7 @@ Parasol Stars
 		<publisher>bootleg (TourVision) / Human</publisher>
 		<info name="alt_title" value="F1トリプルバトル"/>
 		<part name="cart" interface="tourvision_cart">
-			<feature name="id" value="0xff"/>
+			<feature name="id" value="0x64"/> <!-- ID=100, not actually readed from cart -->
```

Comment:
The past tense or "read" is "read" (pronounced differently but spelled identically).  Does this come from reverse-engineering the BIOS, or from finding a gap in the pattern?  It might be worth noting that.

Reply:
Fixed, thanks and sorry!